src/api/app/services/project_configuration_service/project_configuration_presenter.rb

```diff
@@ -0,0 +1,30 @@
+module ProjectConfigurationService
+  class ProjectConfigurationPresenter
+    attr_reader :config
+    def initialize(project, params)
+      @project = project
+      @params = params
```

Comment:
[nitpick]

I think calling `call` here would make it nicer to use. `call` could then be private.

Reply:
@dmarcoux can you reformulate your question. I didn't understand it.

Reply:
The `initialize` method would be like this:
```
def initialize(project, params)
  @project = project
  @params = params
  call
end
```

And then, you would never need to use `call` directly since it's always needed anyway to use `call`.

Reply:
no @dmarcoux IMO that's wrong. Having just call exposed, allows me to mock the service object giving a proc to it, as I'm doing in my specs.. that's the way to go.

Reply:
Good point with mocking :+1: